specs/provisioningInfo.md

```diff
@@ -0,0 +1,94 @@
+
+## Introduction
+The JSON permissions model seeks to be the authoritative source of truth for for data on permissions to API mapping. We have restricted the contents of the JSON permission files to modeling data that has to be reviewed for every change. This does not include data about the permission in the cloud environments where it is intended to be published to. Some of that data is only known after the permission has been published to the environment and would have to either be automatically or manually updated back to the permissions model file after publishing e.g., the GUID of the permission in cloud environment. This should not require a review of the change to the permission file. The provisioning info file is complementary to the JSON permissions model files and allows us to keep the permissions model file as the authoritative source of truth for permissions modeling data while allowing us to maintain a separate source of truth for cloud availability data.
+
+The provisioning info file is a single file for all permissions. Permissions are still the primary concept with a many to many relationship between permissions and cloud environments. Permission claim values are the top level members to enable quick lookup for provisioning for a given permission. Not all implemented permission schemes are published across all environments. The same permission would have different GUIDs in different environments and even when all schemes of a permission are published to the same environment, they would happen to have different unique identifiers. The provisioning info file allows us to model the cloud availability of a permission across different clouds for each of its schemes.
+
+## Notational Conventions
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in BCP 14 [RFC2119] [RFC8174] when, and only when, they appear in all capitals, as shown here.
+
+## <a name="permissionsObject"></a> Permissions JSON Object
+This is the top level object in the provisioning info file. It contains a list of permission claim values and the cloud environments in which they are present.
+
+```json
+"permissions": {
+  “Foo.Read.All”: {
+    "environments": {
+        "global": {
+          "versions": {
+            "v1": {
+                "schemes": [
+                  {
+                    "DelegatedWork": {
+                      "isPresent": true,
+                      "isHidden": false,
+                      "isDisabled": false,
+                      "id": "bed71653-0fa8-42a0-b33c-a1aab02ecda6"
+                    }
+                  }
+                ]
+            },
+            "beta": {
+                "schemes": [
+                  {
+                    "DelegatedWork": {
+                      "isPresent": true,
```

Comment:
This feels redundant. Can we implicitly assume that if we don't have an entry then it's not present?

Reply:
Agreed